internal/deploy/connector/local.go

```diff
@@ -63,10 +63,31 @@ func (exe *localExecutor) cleanContainers() {
 	}
 }
 
+func getController(namespace string) (ctrl config.Controller, err error) {
+	controllers, err := config.GetControllers(namespace)
+	if err != nil {
+		fmt.Print("You must deploy a Controller to a namespace before deploying any Connector")
+		return
+	}
+	if len(controllers) != 1 {
+		return ctrl, util.NewInternalError("Only support 1 controller per namespace")
+	}
+	return controllers[0], nil
+}
+
 func (exe *localExecutor) deployContainers() error {
 	// Deploy connector image
 	util.SpinStart("Deploying Connector")
 
+	controller, err := getController(exe.namespace)
+	if err != nil {
+		return err
+	}
+
+	if util.IsLocalHost(controller.Host) == false {
+		return util.NewInputError("Cannot deploy a local connector with a remote controller")
```

Comment:
Connector and Controller should be capitalized as they are proper nouns. Have done this elsewhere in the code and docs